src/core/etl/tests/Flow/ETL/Tests/Unit/Extractor/FlowTestCase.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Unit\Extractor;
+
+use Flow\ETL\{Config, Extractor, FlowContext, Rows};
+use PHPUnit\Framework\TestCase;
+
+abstract class FlowTestCase extends TestCase
+{
+    public function assertExtractorCountBatches(int $expectedCount, Extractor $extractor) : void
```

Comment:
I think we should make all the methods static and final

Reply:
additionally, to all methods that need to execute Extractor, I would add an extra options argument for FlowContext so in case it's needed it can be configured in some way

Reply:
`assertExtractedBatchesCount`

Reply:
> assertExtractedBatchesCount
Fixed in assertExtractedBatchesCount

> additionally, to all methods that need to execute Extractor, I would add an extra options argument for FlowContext so in case it's needed it can be configured in some way
Fixed in https://github.com/flow-php/flow/pull/1291/commits/7294309d14c409368408f34cd80e132f3e9f1271

> I think we should make all the methods static and final
Fixed in https://github.com/flow-php/flow/pull/1291/commits/466151b9ccdd87ddce3c55305370b51fcf9eb8f8